programs/drift_vaults/src/state/vault.rs

```diff
@@ -115,7 +115,13 @@ pub struct Vault {
     pub cumulative_fuel_per_share: u128,
     /// The total fuel accumulated
     pub cumulative_fuel: u128,
-    pub padding: [u64; 3],
+    pub vault_class: VaultClass,
```

Comment:
opted to put this here instead of using `padding1` above since it looks like replacing `padding1` will break old clients. (anchor client tries to deserialize padding1 which doesnt exist

Reply:
maybe i can use that padding1 above, and just split the remaining padding `padding1` and `padding` so we can maintain optionality with 2 u64s